playbooks/openstack-osh_hostconfig.yml

```diff
@@ -5,12 +5,13 @@
     - name: Ensure all the vars are loaded
       include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
 
-    # TODO(evrardjp): Test if we can include this step in the creation of the node.
-    # In the meantime, we rely on raw ansible module to run task, which causes
-    # the absence of idempotency, and therefore complaining of ansible-lint (301)
     - name: Configure host repositories
-      raw: |
-        zypper ar -f -G {{ deploy_on_openstack_repos_to_configure[item] }} {{ item }}
+      zypper_repository:
```

Comment:
Please check if this is affecting us here https://github.com/ansible/ansible/issues/23257

Reply:
FYI I see Itxaka's testing here: https://github.com/ansible/ansible/issues/23257#issuecomment-463223472